src/Akka.HealthCheck.IHealthCheckExample/HealthChecks/AkkaLivenessActorHealthCheck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Akka.Actor;
+using Akka.HealthCheck.Liveness;
+using Akka.Hosting;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Akka.HealthCheck.IHealthCheckExample.HealthChecks;
+
+public class AkkaLivenessActorHealthCheck<T> : IHealthCheck where T : ActorBase
```

Comment:
I think that it makes sense to put these somewhere in the library (NuGet) so that it's not a copy/paste effort.